frontend/taipy-gui/src/components/Taipy/Chart.tsx

```diff
@@ -616,34 +617,36 @@ const Chart = (props: ChartProp) => {
     return render ? (
         <Box id={id} data-testid={props.testId} className={className} ref={plotRef}>
             <Tooltip title={hover || ""}>
```

Comment:
Just move the tooltip around the parent `Box`

Reply:
Yes I did

Reply:
... like in Chart